lib/systeminstall/pleasewrap.rb

```diff
@@ -8,5 +8,25 @@
 
 #libdir = File.expand_path("../lib", File.dirname(__FILE__))
 #$LOAD_PATH << libdir if File.exist?(File.join(libdir, "pleaserun", "cli.rb"))
+
+require 'open3'
+
+# Work around for https://github.com/elastic/logstash/issues/10593
+# Issue on JRUBY https://github.com/jruby/jruby/issues/5642
+# Workaround retrieved from https://github.com/jruby/jruby/issues/5642#issuecomment-479671017
+if java.lang.System.getProperty("java.version").start_with?("11")
```

Comment:
I suggest we also add a condition on the JRuby version change so that if we update JRuby and forget to verify if this is still needed a least a build failure will direct us to take a look and either remove that or push back the check for the next JRuby version. WDYT?

Reply:
Since there's no JRuby version fixed already and it's not guaranteed the next release will have a fix, what do you suggest to use as a condition here?

Reply:
if the version of JRuby is different than the one in use now then fail? that way if/when we upgrade JRuby it will force us to verify  if patch is still needed and then adjust in either cases, no?

Reply:
I am just trying to avoid keeping code around, especially monkey patches, that might not be needed at some point.

Reply:
done:
```
# dpkg-reconfigure logstash
Using provided startup.options file: /etc/logstash/startup.options
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jruby.util.SecurityHelper to field java.lang.reflect.Field.modifiers
WARNING: Please consider reporting this to the maintainers of org.jruby.util.SecurityHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
RuntimeError: A workaround is in place for JRUBY-5642 that should be applied only to JRuby 9.2.6.0, but found 9.2.3.0
  <main> at /usr/share/logstash/lib/systeminstall/pleasewrap.rb:19
Unable to install system startup script for Logstash.
root@buster:/vagrant#
```

Reply:
Ah!! perfect! 🖤